pootle/static/js/stats.js

```diff
@@ -366,11 +370,37 @@ const stats = {
 
       $('#js-stats-checks').show();
     }
-
     this.hasChecksData = true;
     this.toggleChecksVisibility();
-  }
+  },
+
+  checksStateIsOpen() {
+    return (window.location.search === '?details');
+  },
+
+  checksStatePush() {
+    if (window.history && window.history.pushState) {
+      if (!this.isExpanded) {
+	window.history.pushState("", "", this.pootlePath);
+      } else {
+	window.history.pushState("", "", "?details");
```

Comment:
As a precaution, it's important not to blindly `pushState()`, otherwise extra history entries are created which add no value. So it's necessary to check if the current location differs from the target location before creating unnecessary history entries.

Also note the first parameter to `pushState()` is an object, not a string ([see docs](https://developer.mozilla.org/en-US/docs/Web/API/History/pushState)).